src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs

```diff
@@ -86,37 +82,37 @@ public async Task<CommandHandlingResult> Handle(EnrollToMatchingEngineCommand co
                 throw new InvalidOperationException("ME response is null, don't know what to do");
             }
 
-            if (cashInResult.Status == MeStatusCodes.Ok ||
-                cashInResult.Status == MeStatusCodes.Duplicate)
+            switch (cashInResult.Status)
             {
-                if (cashInResult.Status == MeStatusCodes.Duplicate)
-                {
-                    _log.WriteInfo(nameof(EnrollToMatchingEngineCommand), command.OperationId, "Deduplicated by the ME");
-                }
+                case MeStatusCodes.Ok:
+                case MeStatusCodes.Duplicate:
+                    if (cashInResult.Status == MeStatusCodes.Duplicate)
```

Comment:
i suggest to remove if this way

case Duplicate:
     log(...)
     // fallback to Ok case (without break)

case Ok:
    //proceed publishing event and so on
   ...

Reply:
Discussed in private messages.
Conclusion: we can fall from one `case` to another only when the upper `case` does not contain a code. The trick is using `goto` keyword, but it does not look like a good style.